README.md

```diff
@@ -1,95 +1,132 @@
+- [Developer Documentation](#-developer-documentation)
+- [Configuration & Theming](#-configuration--theming)
+- [ReCaptcha](#-recaptcha)
+- [Community & Support](#-community--support)
+- [Contributing](#-contributing)
+- [Roadmap](#-roadmap)
+- [License](#-license)
 
-- Update to Bootstrap 5 and migration of icons to Bootstrap Icons.
+## 🚀 About
 
-![Update to Bootstrap 5 and migration of icons to Bootstrap Icons](./Web/img/readme/01.png)
+**LibreBooking** is an open-source scheduling solution, forked from Booked Scheduler. It offers a flexible, mobile-friendly, and extensible interface for organizations to manage resource reservations. The repository for the last open source version of Booked Scheduler is maintained here; the `develop` branch contains the latest working code (beta), while `master` is the stable release.
 
-- Design changes.
+## ✨ Features
 
-![Design changes](./Web/img/readme/02.png)
-![Design changes](./Web/img/readme/03.png)
-![Design changes](./Web/img/readme/04.png)
-![Design changes](./Web/img/readme/05.png)
-![Design changes](./Web/img/readme/06.png)
-![Design changes](./Web/img/readme/07.png)
-![Design changes](./Web/img/readme/08.png)
+- [x] Multi-resource booking & waitlists
+- [x] DataTables for advanced listings
+- [x] Role-based access control
+- [x] Quotas and credits for reservations
+- [x] Granular usage reporting
+- [x] Responsive Bootstrap 5 interface
+- [x] Custom themes and color schemes
+- [x] Plugin-ready architecture
+- [x] Outlook/Thunderbird integration through ics
 
-- "Back to top" button on all pages.
+## 📸 Screenshots
 
-!["Back to top" button on all pages](./Web/img/readme/09.png)
+![Login](./Web/img/readme/02.png)
+![Schedules](./Web/img/readme/06.png)
+![Dashboard](./Web/img/readme/03.png)
+![User profile](./Web/img/readme/04.png)
+![Search](./Web/img/readme/07.png)
+![DataTables example](./Web/img/readme/15.png)
 
-- Rename Web/booked.css to Web/librebooking.css (this is no longer Booked Schedule ;-) ).
+## 🔧 Installation & Deployment
 
-- Ability to change the color scheme in `config.php $conf['settings']['css.theme'] = 'default'`  and/or customize an existing one in `Web/css/librebooking.css`
+### Manual Installation
 
-![color scheme](./Web/img/readme/10.png)
-![color scheme](./Web/img/readme/11.png)
-![color scheme](./Web/img/readme/12.png)
-![color scheme](./Web/img/readme/13.png)
-![color scheme](./Web/img/readme/14.png)
+For full setup instructions, see [INSTALLATION.md](./doc/INSTALLATION.md)
 
-- Use of DataTables.
+### Docker Deployment
 
-![Use of DataTables](./Web/img/readme/15.png)
+LibreBooking is available as a Docker container. See [LibreBooking Docker README](https://github.com/LibreBooking/docker) for complete setup.
 
-- Removal of obsolete libraries (e.g. Owl, FloatThead).
+```bash
+git clone https://github.com/LibreBooking/docker.git
+cd docker
+docker-compose up -d
+```
 
-## Prerequisites
+## 💻 Developer Documentation
```

Comment:
Should we keep the "Prerequisites" section? But updated with the real requirements. I do realize that it is up above with the icons, but those are easy to miss in my opinion.